lib/queuePopulator/LogReader.js

```diff
@@ -423,16 +428,29 @@ class LogReader {
                 if (err) {
                     this.log.error(
                         'error publishing entries from log to topic',
-                        { method: 'LogReader._processPublishEntries',
-                          topic,
-                          entryCount: topicEntries.length,
-                          error: err });
+                        {
+                            method: 'LogReader._processPublishEntries',
+                            topic,
+                            entryCount: topicEntries.length,
+                            error: err,
+                        }
+                    );
+                    const labels = Object.assign({},
+                        this.getMetricLabels(),
+                        { publishStatus: 'FAILURE' }
```

Comment:
Do we need constants for the status?

Reply:
When we add status processor status values we should standardize terms, for now I think it is fine.